processes/anomaly_detection.json

```diff
@@ -0,0 +1,1508 @@
+            "name": "date",
+            "schema": {
+                "examples": [
+                    [
+                        "2015-01-01T00:00:00Z",
+                        "2016-01-01T00:00:00Z"
+                    ],
+                    [
+                        "2015-01-01",
+                        "2016-01-01"
+                    ]
+                ],
+                "items": {
+                    "anyOf": [
+                        {
+                            "format": "date-time",
+                            "subtype": "date-time",
+                            "type": "string"
+                        },
+                        {
+                            "format": "date",
+                            "subtype": "date",
+                            "type": "string"
+                        },
+                        {
+                            "maxLength": 4,
+                            "minLength": 4,
+                            "pattern": "^\\d{4}$",
+                            "subtype": "year",
+                            "type": "string"
+                        },
+                        {
+                            "type": "null"
+                        }
+                    ]
+                },
+                "maxItems": 2,
+                "minItems": 2,
+                "subtype": "temporal-interval",
+                "type": "array"
+            }
+        },
+        {
+            "description": "Loaded feature collection object of the field geometries",
+            "name": "polygon",
+            "schema": {
+                "subtype": "geojson",
+                "type": "object"
+            }
+        },
+        {
+            "default": "FAPAR",
+            "description": "BIOPAR type [FAPAR,FCOVER] used to calculate the CropSAR curve on. As default the FAPAR is used",
+            "name": "biopar_type",
+            "optional": true,
+            "schema": {
+                "type": "string"
+            }
+        }
+    ],
+    "returns": {
+        "description": "A file with the CropSAR fAPAR values of the individual fields and the regional average fAPAR value for the specified time period",
+        "schema": {
+            "type": "object"
+        }
+    },
+    "process_graph": {
+        "adddimension1": {
+            "arguments": {
+                "data": {
+                    "from_node": "reducedimension1",
+                    "node": {
+                        "arguments": {
+                            "data": {
+                                "from_node": "resamplecubespatial1",
+                                "node": {
```

Comment:
@m-mohr For this openeo process, they perform some mandatory calculations before using it as a final process. I guess that is represented inside a node. I confirmed with my colleague that it should be one of the steps for this process, and we are not sure if removing it will give the desired outcome.

Reply:
I really meant the JSON key "node" here, not an openEO node in the process graoh.  The key "node" is undefined alongside a "from_node" and should be removed.

Reply:
@m-mohr, the process graph is updated.

Reply:
Great, thanks.